PowerPointLabs/PowerPointLabs/TextCollection/CommonText.cs

```diff
@@ -3,12 +3,16 @@
     public class CommonText
     {
         # region Common Error
+        public const string ErrorCaption = "Error";
+        public const string ErrorSlideSelectionTitle = "Slide selection error";
+        public const string ErrorDuringSetup = "Error During Setup.";
+
         public const string ErrorNameTooLong = "The name's length cannot be more than 255 characters.";
         public const string ErrorInvalidCharacter = "The name cannot be empty, or contain the following characters:'<', '>', ':', '\"', '/', '\\', '|', '?', or '*'.";
-        public const string ErrorFileNameExist = "A file already exists with that name.";
-        # endregion
+        public const string ErrorFileNameExist = "A file already exists with that name.";   
+        #endregion
```

Comment:
Why did you remove the spacebar?

Reply:
I will change this asap

Reply:
this is very strange, cause I did not add any spacebar when I made edits to this file